data/init.sql

```diff
@@ -42,7 +42,7 @@ CREATE TABLE IF NOT EXISTS wirelessconfigs(
   link_policy int[],
   creation_date timestamp,
   created_by varchar(40),
-  tenant_id varchar(36) NOT NULL, 
+  tenant_id varchar(36) NOT NULL,
```

Comment:
Make sure you open a PR for OpenAMTCloudToolkit main repo to update the sql scripts both for docker compose and kubernetes.